### DISCUSSION TITLE
[Feature request] Make it possible to detect if espanso is disabled or enabled.

### DISCUSSION THREAD

**Discussion by matheusfillipe** [espanso/espanso#1200] [Feature requests and Ideas]
Currently I can enable or disable espanso, with the espanso daemon running on linux, with the commands `espanso cmd disable` and `espanso cmd enable` or `espanso cmd toggle`. Those have the same effect as toggling with the default shortcut Alt-Alt. 

The problem is that there is no command to know if espanso is enabled or disabled. Would be nice to have something like `espanso cmd status` that would report its state. For now `espanso status` only reports about the daemon state and not espanso itself.

Here is a example:

``` 
$ espanso status
espanso is running

$ espanso cmd disable 

$ espanso status     
espanso is running

## Bellow is what I think would be a nice feature
# $ espanso cmd status
# espanso is disabled

$ systemctl stop espanso --user 

$ espanso status               
espanso is not running

$ systemctl start espanso --user

$ espanso status                
espanso is running
```

**Comment by federico-terzi** [comment]
@matheusfillipe Thank you! I'll add it to the roadmap :)
